system-addon/content-src/components/Topics/_Topics.scss

```diff
@@ -39,11 +40,9 @@
     color: $link-secondary;
 
     @media (min-width: $break-point-large) {
-      float: right;
-
-      &:dir(rtl) {
-        float: left;
-      }
+      position: absolute;
+      top: 0;
+      offset-inline-end: 0px;
```

Comment:
The `float` here was to allow for long topics to wrap correctly without overlapping the link (just in case some locales have longg topics). Here's with this change:

<img width="762" alt="image" src="https://user-images.githubusercontent.com/438537/31361144-b22f95d2-ad06-11e7-9491-6056d8e2b7dc.png">

Reply:
gah. hate floats. they change size depending on if the parent is overflow visible or not. ok. maybe flex? or maybe force the height somehow.